installinator/src/peers.rs

```diff
@@ -359,7 +359,7 @@ struct ArtifactClient {
     // TODO: this currently uses a wicketd client. However, because the standard wicketd server is
```

Comment:
Can probably remove this todo at this point.

Reply:
Done.